doc/api/worker_threads.md

```diff
@@ -61,6 +61,35 @@ Worker threads inherit non-process-specific options by default. Refer to
 [`Worker constructor options`][] to know how to customize worker thread options,
 specifically `argv` and `execArgv` options.
 
+## `worker.environmentData`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Within a worker thread, `worker.environmentData` is a {Map} containing a clone
```

Comment:
This allows code in the worker to iterate, delete and clear all keys.  I'm guessing at some point someone will write code that removes data which it doesn't own.  Not sure anything should change here, just making this comment so others can consider it.

Reply:
Yep. FWIW, `workerData` has precisely the same "issue" and I was intentionally going for a similar approach.